tests/test_dpa.py

```diff
@@ -0,0 +1,18 @@
+import kubernetes
+
+from openshift.dynamic import DynamicClient
+from openshift_resources.data_protection_application import DataProtectionApplication
+
+
+class DynamicClient:
+    pass
+
+
+def oadp_ocp_client():
```

Comment:
why not passing the client to the resources? for example, passing the client to the DataProtectionApplication object, and then using it throughout the test?
maybe we should also consider do this initialization as as session scoped hook? to avoid initializing it for each test, and then just retrieving it as a singleton or something...

Reply:
RE: I see NamespacedResource already has client argument, so we could pass it with super().\_\_init\_\_

Reply:
yes you can pass the client

Reply:
it is supposed to be a fixture, it yields a new client
and we can set the scope.
I hope the addresses it